src/Fields/field_tuples.jl

```diff
@@ -205,16 +205,13 @@ may be specified via a named tuple of `FieldBoundaryCondition`s.
 """
 function PressureFields(grid, bcs=NamedTuple())
```

Comment:
Yeah, agreed. I'm first just trying to do the bare minimum while this is a draft PR so that I can see what else needs to be changed while I can see test the branch on my code. Later when I have more time I'll try to do the tidying up if no one has done that yet.